pkg/cloud/google/machineactuator.go

```diff
@@ -236,19 +236,19 @@ func (gce *GCEClient) ProvisionClusterDependencies(cluster *clusterv1.Cluster) e
 	return gce.serviceAccountService.CreateMasterNodeServiceAccount(cluster)
 }
 
-func (gce *GCEClient) Create(cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
+func (gce *GCEClient) Create(ctx context.Context, cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
```

Comment:
Is ctx unused (for now)? If so, maybe make it `_ context.Context` to make that more obvious to the reader.

Reply:
Ooops - missed this comment before.  I just implemented it.

(Although I think we should be passing the context around to e.g. Instances.Get(), now that we have it ... but that's a separate PR)